notes-webapp/src/main/webapp/vue-app/note-page-view/components/NoteRichEditor.vue

```diff
@@ -123,6 +123,8 @@ export default {
             offset: 40,
             unit: 'px'
           },
+          autoGrow_onStartup: true,
+          autoGrow_maxHeight: 800,
```

Comment:
Should be able to change this in the dedicated file of configuration:

https://github.com/Meeds-io/notes/blob/83e6286aec1054f9f614d7b3d96ad542b0ffe847/notes-webapp/src/main/webapp/WEB-INF/conf/wiki/ckeditor/inline-config.js#L55-L56

Reply:
ok for me, thanks. I resolve this comment and let you continue the review with the team.